FpsCheck.py

```diff
@@ -93,7 +93,7 @@ def load_sets(self):
         self.error_list += load_sets_errors
         return check_sets
 
-    def has_all_rationales(self, check_sets):
+    def has_all_rationales(self):
```

Comment:
I don't quite follow what this change has to do with the diffs - do we always validate all of the rationales, even on sets that haven't changed?

Reply:
This change was made because how has_all_rationales previously worked was causing issues with the change made by implementing the diff changes (it has to do with rationales not being saved as part of the FpsSet object). I reworked the function in the recent commit that fix the way it works with the diff changes, but still take the check_sets argument.